workflow/tests/base_results/results.csv

```diff
@@ -126,8 +126,8 @@ base-enclosure-split-surfaces2.xml,36.18,36.18,16.88,0.0,0.0,0.0,0.0,0.0,0.0,0.4
-base-enclosure-windows-physical-properties.xml,35.38,35.38,22.26,0.0,0.0,0.0,0.0,0.0,0.0,0.58,3.65,0.71,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,22.26,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,21.05,11.97,9.23,0.61,0.0,0.0,0.0,4.0,2178.0,3775.0,25.26,21.39,0.0,3.83,3.99,0.55,4.06,0.72,5.86,0.0,0.0,3.02,-0.2,5.06,0.0,0.77,0.0,4.96,-11.67,0.0,0.1,-0.33,0.0,3.93,0.01,8.35,0.0,0.0,-5.93,-0.15,-0.9,-5.84,-0.13,0.0,3.2,9.62,1355.0,998.0,11400.0,2616.0
-base-enclosure-windows-shading.xml,32.75,32.75,17.17,0.0,0.0,0.0,0.0,0.0,0.0,0.45,1.64,0.22,9.17,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,17.17,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,16.23,3.62,9.23,0.62,0.0,0.0,0.0,0.0,2132.0,2584.0,20.48,12.41,0.0,3.98,4.07,0.56,4.04,0.73,2.23,0.0,0.0,3.0,-0.17,5.16,0.0,0.79,0.0,3.83,-12.07,0.0,0.62,0.09,0.04,4.05,0.11,-0.91,0.0,0.0,-5.62,-0.12,-0.68,-4.13,-0.08,0.0,1.0,9.22,1355.0,998.0,11400.0,2616.0
+base-enclosure-windows-physical-properties.xml,35.68,35.68,21.6,0.0,0.0,0.0,0.0,0.0,0.0,0.56,3.91,0.77,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,21.6,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,20.43,13.09,9.23,0.61,0.0,0.0,0.0,4.0,2180.0,3445.0,25.1,21.5,0.0,3.84,3.99,0.55,4.07,0.72,5.28,0.0,0.0,3.05,-0.19,5.06,0.0,0.77,0.0,4.83,-11.63,0.0,0.05,-0.37,0.0,3.92,0.0,9.56,0.0,0.0,-5.96,-0.14,-0.92,-6.07,-0.14,0.0,3.48,9.66,1355.0,998.0,11400.0,2616.0
+base-enclosure-windows-shading.xml,33.02,33.02,15.72,0.0,0.0,0.0,0.0,0.0,0.0,0.41,1.89,0.28,9.17,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.72,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.86,4.63,9.23,0.62,0.0,0.0,0.0,0.0,2151.0,2612.0,20.49,12.67,0.0,3.93,4.02,0.55,3.99,0.72,0.92,0.0,0.0,3.06,-0.14,5.07,0.0,0.77,0.0,3.56,-11.69,0.0,0.41,-0.1,0.01,3.96,0.06,0.76,0.0,0.0,-5.72,-0.1,-0.85,-4.63,-0.11,0.0,1.3,9.6,1355.0,998.0,11400.0,2616.0
```

Comment:
I wouldn't think the results should change for these files. It seems like different windows are now being assigned different properties/shading.